### PR TITLE
Interacctive list - Second Milestone

### PR DESCRIPTION
**Pull request by benwmx**
### Details 🔢 
the requirement fulfilled in this milestone : 
* Add new JavaScript functions and import them as modules.
* Add event Listeners to the checkbox.
* Add local storage to save the changes.

## ScreenShot
demo

### CODE REVIEW

File: src/getStorage.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export default function getStorage() {
+  const storage = JSON.parse(localStorage.getItem('storage'));
+  return (storage !== null || storage !== undefined) ? storage : [];
+}
```
Comment (hemantso) [inline src/getStorage.js]: - [ ] As per project requirements we have to store the updated array of items in local storage, so the user gets the correct list values after the page reloads. But after marking the task as completed and reload the page than it comes to the original position. Kindly try to fix this issue when the user refresh the page it should render updated array item

File: src/showList.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  const listDiv = document.getElementById('list');
+  listDiv.innerHTML = '';
+  for (let index = 0; index < tasksList.length; index += 1) {
+    const task = document.createElement('li');
+    const remove = document.createElement('i');
+    const description = document.createElement('p');
+    if (tasksList[index].completed) {
+      const checked = document.createElement('i');
+      checked.className = 'fas fa-check';
+      description.appendChild(checked);
```
Comment (hemantso) [inline src/showList.js]: - [ ] As per the learning objective, We have to use proper ES6 syntax. Kindly try to use the arrow function as per ES6 guidlines.

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @benwmx , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benwmx** [comment]
thank you 🙏🏽  @hemant-soni-vst-au4  for your review, I fixed the problems that you pointed to.

**Review by ZawadiSumba66 — APPROVED** [review]
Hi @benwmx 👋 , 
## STATUS: APPROVED 👍 
Great job implementing changes requested by the previous reviewer. 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**